### PR TITLE
Updated intro section.

### PR DESCRIPTION
**Pull request by metasim**
Added additional raster-read section.

### CODE REVIEW

File: pyrasterframes/src/main/python/docs/raster-read.pymd [hunk 1]
```diff
@@ -55,6 +53,64 @@ tile = rf.select(rf_tile("proj_raster")).first()[0]
 display(tile)
 ```
 
+## Multiple Singleband Rasters
+
+In this example, we show reading [two bands](https://en.wikipedia.org/wiki/Multispectral_image) of [Landsat 8](https://landsat.gsfc.nasa.gov/landsat-8/) imagery (red and near-infrared), combining them with `rf_normalized_difference` to compute NDVI. As described in the section on @ref:[catalogs](raster-catalogs.md), image URIs in a single row are assumed to be from the same scene/granule, and therefore compatible. This pattern is commonly used when multiple bands are stored in separate files.
```
Comment (vpipkt) [inline pyrasterframes/src/main/python/docs/raster-read.pymd]: Possibly link internal to concepts.html#band ?

Comment (vpipkt) [reply pyrasterframes/src/main/python/docs/raster-read.pymd]: And also maybe link to some external resource about NDVI.  Consider a parenthetical like, `[NDVI](some-link.html), a common measure of plant vigor.`.

File: pyrasterframes/src/main/python/docs/raster-read.pymd [hunk 2]
```diff
@@ -55,6 +53,64 @@ tile = rf.select(rf_tile("proj_raster")).first()[0]
...
+When reading a multiband raster or a @ref:[_catalog_](#raster-catalogs) describing multiband rasters, you will need to know ahead of time which bands you want to read. You will specify the bands to read, **indexed from zero**, as a list of integers into the `band_indexes` parameter of the `raster` reader.
+
+For example, we can read a four-band (red, green, blue, and near-infrared) image as follows. The individual rows of the resulting DataFrame still represent distinct spatial extents, with a projected raster column for each band specified by `band_indexes`.
+
+```python, multiband
+mb = spark.read.raster(
+    's3://s22s-test-geotiffs/naip/m_3807863_nw_17_1_20160620.tif',
+    band_indexes=[0, 1, 2, 3],
+)
+display(mb)
```
Comment (vpipkt) [inline pyrasterframes/src/main/python/docs/raster-read.pymd]: this looks great rendered

File: pyrasterframes/src/main/python/docs/raster-read.pymd [hunk 3]
```diff
@@ -55,6 +53,64 @@ tile = rf.select(rf_tile("proj_raster")).first()[0]
...
+In this example, we show reading [two bands](https://en.wikipedia.org/wiki/Multispectral_image) of [Landsat 8](https://landsat.gsfc.nasa.gov/landsat-8/) imagery (red and near-infrared), combining them with `rf_normalized_difference` to compute NDVI. As described in the section on @ref:[catalogs](raster-catalogs.md), image URIs in a single row are assumed to be from the same scene/granule, and therefore compatible. This pattern is commonly used when multiple bands are stored in separate files.
+
+```python, multi_singleband
+bands = [f'B{b}' for b in [4, 5]]
+uris = [f'https://landsat-pds.s3.us-west-2.amazonaws.com/c1/L8/014/032/LC08_L1TP_014032_20190720_20190731_01_T1/LC08_L1TP_014032_20190720_20190731_01_T1_{b}.TIF' for b in bands]
+catalog = ','.join(bands) + '\n' + ','.join(uris)
+
+rf = spark.read.raster(catalog, bands) \
+    .withColumnRenamed('B4', 'red').withColumnRenamed('B5', 'NIR') \
+    .withColumn('longitude_latitude', st_reproject(st_centroid(rf_geometry('red')), rf_crs('red'), lit('EPSG:4326'))) \
```
Comment (vpipkt) [inline pyrasterframes/src/main/python/docs/raster-read.pymd]: maybe worth breaking this out with a few more code comments explaining what we are doing with each column.

File: pyrasterframes/src/main/python/docs/index.md [hunk 4]
```diff
@@ -2,15 +2,21 @@
...
 
-To learn more, please see the @ref:[Getting Started](getting-started.md) section of this manual.
+![RasterFrames](static/rasterframes-pipeline-nologo.png)
 
-The source code can be found on GitHub at [locationtech/rasterframes](https://github.com/locationtech/rasterframes).
+Through its custom [Spark DataSource](https://rasterframes.io/raster-read.html), RasterFrames can read various raster formats -- including GeoTIFF, JP2000, MRF, and HDF -- and from an [array of services](https://rasterframes.io/raster-read.html#uri-formats), such as HTTP, FTP, HDFS, S3 and WASB. It also supports reading the vector formats GeoJSON and WKT/WKB. RasterFrame contents can be filtered, transformed, summarized, resampled, and rasterized through [200+ raster and vector functions](https://rasterframes.io/reference.html).
```
[the remark this answers is not in the document]
Comment (metasim) [reply pyrasterframes/src/main/python/docs/index.md]: In the new heterogeneous CRS regime, RasterFrame is basically anything that has tile columns, so I think it's ok to keep this (and promote the name).

**Pull request merged**